nextflow.config

```diff
@@ -12,15 +12,33 @@ params {
   // TODO nf-core: Specify your pipeline's command line flags
   genome = false
   reads = false
+  readPaths = false
   single_end = false
   outdir = './results'
 
+  bam = false
+  bed = false
+  bai = false
+
+  skip_trimming = false
+
+  csv_protein_fasta = false
   protein_fastas = false
   hashes = false
+  hash2kmer_ksize = 21
+  hash2kmer_molecule = 'protein'
+
+  // Differential hash expression options
+  diff_hash_expression = false
+  diff_hash_inverse_regularization_strength = 0.1  // Small numbers for fewer features
+  diff_hash_solver = 'saga'  // Saga solver is fast for large datasets
+  diff_hash_penalty = 'l1'   // Use strong penalty for large datasets
 
   extract_coding_peptide_molecule = "protein"
   // UNIPROT human proteome is default reference. Human has Taxon ID 9606
   extract_coding_peptide_fasta = "ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/reference_proteomes/Eukaryota/UP000005640_9606.fasta.gz"
+  extract_coding_jaccard_threshold = 0.8
```

Comment:
Is it okay to link to a review out of github ? What if its moved? I am not sure if it would be ever just curious